test/_requestToken.js

```diff
@@ -71,7 +71,7 @@ describe('_requestToken', function() {
 	it('should fail if request module fails', function(done) {
 		// subsequently, this will test merging options as well
 		AuthClient
-			._requestToken({ method: 'TEST' })
```

Comment:
@vernak2539 , I went ahead and just reverted the removal of the method for you, so we're not touching this test, since, really, it's unrelated to this pull request.